### PR TITLE
doc: add faq over environments which are restricted for external access

### LINKED ISSUE #2888: goproxy for operator sdk new operator creation.
## Type of question

general context question on the goproxy for building a new operator.

## Question

without setting goproxy 'https://proxy.golang.org,direct '
we are unable to build the new operator. 
My environment is restricted for external access.
need inputs on such environments for building operators without
setting goproxy. 
need info on alternate ways of building without goproxy and what are we fetching via goproxy for building a new operator.

Thanks in advance.

**Environment**
* operator-sdk version:
"v0.16.0"
  insert release or Git SHA here

* Kubernetes version information:
Client Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.4", GitCommit:"224be7bdce5a9dd0c2fd0d46b83865648e2fe0ba", GitTreeState:"clean", BuildDate:"2019-12-11T12:47:40Z", GoVersion:"go1.12.12", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.4", GitCommit:"224be7bdce5a9dd0c2fd0d46b83865648e2fe0ba", GitTreeState:"clean", BuildDate:"2019-12-11T12:37:43Z", GoVersion:"go1.12.12", Compiler:"gc", Platform:"linux/amd64"}
  insert output of `kubectl version` here

* Kubernetes cluster kind: 

**Additional context**
Add any other context about the question here.

### PR DESCRIPTION
**Pull request by camilamacedo86**
**Description of the change:**
- Add info over how to solve the problem to get the dependencies when environments are restricted for external access

**Motivation for the change:**
Closes #2888

### CODE REVIEW

**Comment by joelanford** [comment]
IMO, this is a general Go programming issue. I'm not sure it makes sense to document this in the SDK repo since this isn't something the SDK itself can really help with. 

This also seems prone to bit rot. Will we be able to keep this up-to-date if the Go community makes changes related to supporting disconnected environments?

File: website/content/en/docs/faq.md [hunk 1]
```diff
@@ -129,6 +129,24 @@ func (r *ReconcileMemcached) Reconcile(request reconcile.Request) (reconcile.Res
 	}
 ```
 
+## My environment is restricted for external access and I am unable to build the operators, how do I fix this?
+
+To create the project is required to have its dependencies. So, because of this, the `GOPROXY` environment variable with 'https://proxy.golang.org,direct' is set by default in go 1.13+. See the [Module downloading and verification](https://golang.org/cmd/go/#hdr-Module_downloading_and_verification) to understand it better.
+
+In this way, to attend the requirements defined in restrictive environments, is possible to:
```
Comment (jberkhahn) [inline website/content/en/docs/faq.md:136]: i think this should be "to attain the defined requirements"?

**Comment by jberkhahn** [comment]
Yeah, I agree with Joe. Isn't go vendoring already obsolete in favor of go modules?

File: website/content/en/docs/faq.md (hunk 1 above)
Comment (camilamacedo86) [inline website/content/en/docs/faq.md:136]:
```suggestion
In this way, to attain the defined requirements in restrictive environments is possible to:
```

**Comment by jmrodri** [comment]
I agree with @joelanford comments as well, we shouldn't document this in the SDK repo. I think we should just close this PR. We can answer it in the issue and that will have to serve as the documentation.

**Comment by camilamacedo86** [comment]
Closing it based in the above comments.

**Pull request closed without merging**